docs/source/services/timelock_service/service-management.rst

```diff
@@ -0,0 +1,70 @@
+.. _timelock-service-management:
+
+==================
+Service Management
+==================
+
+Namespace Changes
+=================
+
+.. danger::
+
+   Be careful when changing namespaces - doing this process incorrectly may result in **SEVERE DATA CORRUPTION**.
+   This guide also **ONLY** covers the steps for migrating a TimeLock client - AtlasDB clients, separately, must also
+   migrate the data in their key-value services to the new namespace.
+
+When a TimeLock client wants to switch its namespace (while preserving its data), care must be taken to ensure that the
+old timestamp bound is carried over to the new namespace. This is to preserve the guarantee of the timestamp service
+that timestamps must be increasing.
+
+Step 1: Shut Down Clients
+-------------------------
+
+Shut down all instances of the TimeLock client.
+This is done to ensure that the timestamp bound does not advance further while we are doing the other step.
```

Comment:
nit: steps

Reply:
also "of the TimeLock client for which you wish to change the namespace". I can see people turning off TimeLock itself by accident

Reply:
Sure, put in both changes. Also added another line telling people to explicitly `Leave TimeLock running; you'll need it for the remaining steps.`